src/BLEBondStore.cpp

```diff
@@ -1,10 +1,14 @@
 // Copyright (c) Sandeep Mistry. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+// Contributors:
+//    Bosch Software Innovations GmbH - Please refer to git log
+
```

Comment:
I'm not to keen on starting the trend of listing contributors per file. It's not something I've seen in most open source projects. Github shows a list of contributors here: https://github.com/sandeepmistry/arduino-BLEPeripheral/graphs/contributors